qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/pairing_table_widget/pairing_table_widget_presenter.py

```diff
@@ -13,22 +13,16 @@
 from mantidqtinterfaces.Muon.GUI.Common.grouping_tab_widget.grouping_tab_widget_model import RowValid
 from mantidqtinterfaces.Muon.GUI.Common.grouping_table_widget.grouping_table_widget_presenter import row_colors, row_tooltips
 
-
-pair_columns = ["pair_name", "to_analyse", "group_1", "group_2", "alpha"]
+from mantidqtinterfaces.Muon.GUI.Common.pairing_table_widget.pairing_table_constants import get_pair_columns, pair_columns
 
 
 class PairingTablePresenter(object):
     def __init__(self, view, model):
         self._view = view
         self._model = model
 
-        self._view.on_add_pair_button_clicked(self.handle_add_pair_button_checked_state)
-        self._view.on_remove_pair_button_clicked(self.handle_remove_pair_button_clicked)
-
-        self._view.on_user_changes_pair_name(self.validate_pair_name)
-        self._view.on_user_changes_alpha(self.validate_alpha)
-        self._view.on_guess_alpha_clicked(self.handle_guess_alpha_clicked)
-        self._view.on_table_data_changed(self.handle_data_change)
+        self._view.subscribe(self)
+        self._view.notify()
```

Comment:
Thanks for adding this two function, it looks good. 
Perhaps we could rename the function `notify` to be `subscribe_notifiers_to_presenter` or something of the sort.